alacritty/src/renderer/mod.rs

```diff
@@ -359,30 +360,46 @@ impl GlyphCache {
     }
 }
 
+// XXX these flags must be in sync with their usage in text.*.glsl shaders.
```

Comment:
```suggestion
// NOTE: These flags must be in sync with their usage in the text.*.glsl shaders.
```

I find `NOTE` a bit nicer than the unspecific `XXX`, should be highlighted just the same in vim at least.